src/Hipace.cpp

```diff
@@ -658,37 +664,31 @@ Hipace::PredictorCorrectorSolveOneSubSlice (const int lev, const int step,
                                             const int islice, const int islice_local)
 {
     m_fields.setVal(0., lev, WhichSlice::This,
-        "ExmBy", "EypBx", "Ez", "Bx", "By", "Bz", "jx", "jy", "jz", "rho", "Psi");
+        "ExmBy", "EypBx", "Ez", "Bx", "By", "Bz", "jx", "jy", "jz", "rhomjz", "Psi");
     if (m_use_laser) {
         m_fields.setVal(0., lev, WhichSlice::This, "chi");
     }
+    if (m_deposit_rho) {
+        m_fields.setVal(0., lev, WhichSlice::This, "rho");
+    }
 
-    // deposit jx jy jz rho and maybe chi
+    // deposit jx jy jz maybe chi and rhomjz
```

Comment:
```suggestion
    // deposit jx jy jz rhomjz and maybe chi
```
Here `rhomjz` is always deposited

Reply:
I want to keep the order of the comment the same as the true true false false true false in the function

Reply:
What about  `jx jy jz (maybe chi) and rhomjz`? otherwise it sounds like maybe `rhomjz`is deposited

Reply:
Yes thats what I put there